README.md

```diff
@@ -1,5 +1,24 @@
 # Northcoders News API
 
+https://alice-nc-news.herokuapp.com/
+
+This project is a RESTful API for accessing news articles and associated comments. Various HTTP requests can be performed to endpoints for articles, comments and users.
+
+# Working with the API
+
+This project can be forked and cloned from GitHub here: https://github.com/althrntn/Alice-BE-News-Project/tree/main
+
+Once cloned, several dependencies are needed to run the project: please install dotenv, express and postgres (pg). For testing purposes, jest, jest-sorted, jest-extended and supertest are also required.
```

Comment:
So as they have the package.json, they can run `npm install` and it will install everything they need